psalm-baseline.xml

```diff
@@ -695,39 +652,35 @@
       <code><![CDATA[$config]]></code>
     </MixedAssignment>
   </file>
-  <file src="src/Helper/ViewModel.php">
-    <PossiblyUnusedMethod>
-      <code><![CDATA[hasRoot]]></code>
-    </PossiblyUnusedMethod>
+  <file src="src/Helper/Service/LayoutFactory.php">
+    <PossiblyUnusedReturnValue>
+      <code><![CDATA[Layout]]></code>
+    </PossiblyUnusedReturnValue>
   </file>
   <file src="src/HelperPluginManager.php">
-    <DeprecatedInterface>
-      <code><![CDATA[null|ConfigInterface|ContainerInterface]]></code>
-    </DeprecatedInterface>
+    <DeprecatedProperty>
+      <code><![CDATA[$this->initializers]]></code>
+      <code><![CDATA[$this->initializers]]></code>
+      <code><![CDATA[$this->initializers]]></code>
+      <code><![CDATA[$this->initializers]]></code>
+    </DeprecatedProperty>
     <DocblockTypeContradiction>
       <code><![CDATA[! $events]]></code>
       <code><![CDATA[! $events]]></code>
     </DocblockTypeContradiction>
+    <MethodSignatureMismatch>
```

Comment:
Can this mismatch be fixed?

Reply:
No, well, maybe. Probably removing the PSR\Container stub will work at the expense of inference there. Also, overriding `get` with repeated doc blocks in the plugin manager will work - it was like that before.